main/2020/CG-08-04.md

```diff
@@ -25,6 +25,10 @@ Installation is required, see the calendar invite.
 1. Adoption of the agenda
 1. Proposals and discussions
     1. Review of action items from prior meeting.
+    1. Continuations & effect handlers as typed stacks ([issue](https://github.com/WebAssembly/design/issues/1359
+)) (Daniel Hillerström, Daan Leijen, Sam Lindley, Matija Pretnar, Andreas Rossberg, KC Sivaramakrishnan) [20 min]
+       - Poll: Phase 1?
```

Comment:
How does phase 1 work given there is another approach being discussed - would both approaches be discussed inside a single repo, sort of like GC is doing? Or would there be separate repos?